README.md

```
@@ -0,0 +1,73 @@
+## 🌐 Architecture
+![image](https://github.com/user-attachments/assets/31c1a8aa-67e4-423d-8d34-05c04bb7f8f8)
+
+</br>
+</br>
+
+## 🛠️ 기술 스택
+### ⚡️Language & Framework
+![springboot](https://img.shields.io/badge/spring&nbsp;boot-%236DB33F.svg?style=for-the-badge&logo=springboot&logoColor=white)
+![springcloud](https://img.shields.io/badge/spring&nbsp;cloud-%236DB33F.svg?style=for-the-badge&logo=springcloud&logoColor=white)
+
+### 💾 Database 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)